src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    autocomplete="off"
+  >
+      <fieldset class="fieldset person-info">
+        <legend>Personal information</legend>
+        <label class="form-field surname">
+        Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+
+        <label class="form-field name">
+        Name:
+          <input
+            type="text"
+            name="name"
+            required
+          >
+        </label>
+
+        <label class="form-field how-old">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form-field date-birth">
+          Full date of birth:
+          <input
+            type="date"
+            name="trip-start"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          <input type="checkbox" name="term">
+          I accept the term of the agreement
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset registration">
+        <legend>Registration</legend>
+
+        <label class="form-field email">
+        E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field password">
+        Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="30"
+            pattern="\w+"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset an-inter-fact">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field cats">
+            Do you love cats?
```

Comment:
move `Do you love cats?` text out the label, because now when you click on it, `yes `becomes selected